### DISCUSSION TITLE
Why can't the useController hook support the deps attribute?

### DISCUSSION THREAD

**Discussion by GuoXiaoyang** [react-hook-form/react-hook-form#11367] [Q&A]
The register() API can support the deps attribute, but it's not possible within useController. We need the deps functionality.
 The feature request has been referred to in this issue. https://github.com/react-hook-form/react-hook-form/issues/8170

**Comment by ktalalykin** [comment]
is it possible with useController
`useController({rules: {deps: <list of your deps>} })`
